src/data/file.cpp

```diff
@@ -67,7 +68,7 @@ void File::addErrorMessage(const ErrorMessage &message)
 
 void File::clearReferences()
 {
-    m_typeReferences.clear();
+    m_referencesMap.clear();
```

Comment:
I believe clearing `m_references` might be needed as well.

Reply:
true, fixing